lib/model.js

```diff
@@ -228,6 +228,12 @@ class Model {
         throw new Error(`A column called 'id' was added to the attributes of '${this.tableName}' but not marked with 'primaryKey: true'`);
       }
 
+      if (!this.options.generatePrimaryKey) {
+        // don't do anything further or it will
+        // implement an id column that we don't want
+        return;
```

Comment:
This also means timestamp, version and paranoid attribute will be disabled when you turn on this flag